src/main/scala/fi/oph/kouta/client/HttpClient.scala

```diff
@@ -27,6 +29,18 @@ trait HttpClient {
       case (xxx, _, response) => errorHandler(url, xxx, response)
     }
 
+  def post[T](url: String, data: String, errorHandler: (String, Int, String) => Nothing = defaultErrorHandler, followRedirects: Boolean = false)(parse: String => T): T = {
+    DefaultHttpClient.httpPost(url, Some(data), defaultOptions(followRedirects):_*)
```

Comment:
Tätä vielä hieman korjaisin, kun nyt oletetaan, että kutsuja tietää, että datan pitää olla json ja kutsujan pitää myös itse serialisoida data jsoniksi. Tämän voisi korjata niin, että content type ja accept-headerit on parametreina, mutta itse mieluummin korjaisin sen näin:

```
def postAsJson[A,T](url: String, data: A, errorHandler: (String, Int, String) => Nothing = defaultErrorHandler, followRedirects: Boolean = false)(parse: String => T): T = {
    DefaultHttpClient.httpPost(url, Some(write[A](data)), defaultOptions(followRedirects):_*)
```

Sitten pitää lisäksi olla
```
trait HttpClient extends KoutaJsonFormats {
```